x-pack/plugins/apm/server/routes/rum_client/get_page_view_trends.ts

```diff
@@ -8,7 +8,13 @@
 import { getRumPageLoadTransactionsProjection } from '../../projections/rum_page_load_transactions';
 import { mergeProjection } from '../../projections/util/merge_projection';
 import { SetupUX } from './route';
-import { BreakdownItem } from '../../../typings/ui_filters';
+
+export interface BreakdownItem {
```

Comment:
is it possible to move this entire folder (`rum_client/`) out of the apm plugin?

Reply:
Yes, this will be a follow up.